ultralytics/engine/tuner.py

```diff
@@ -179,7 +182,14 @@ def __call__(self, model=None, iterations=10, cleanup=True):
         t0 = time.time()
         best_save_dir, best_metrics = None, None
         (self.tune_dir / "weights").mkdir(parents=True, exist_ok=True)
-        for i in range(iterations):
+
+        previous_iteration_count = 0
+
+        if self.tune_csv.exists():
+            x = np.loadtxt(self.tune_csv, ndmin=2, delimiter=",", skiprows=1)
+            previous_iteration_count = len(x)
+
+        for i in range(previous_iteration_count, iterations + previous_iteration_count):
```

Comment:
I felt this was the least intrusive way to handle previous iterations. You could also warn the user or throw an error if previous >= requested iterations.